source/upgrade-guide/legacy/upgrading-elastic-stack/from-6.x-to-6.8.rst

```diff
@@ -168,7 +168,7 @@ Upgrading Logstash
 
           # cp /etc/logstash/conf.d/01-wazuh.conf /backup_directory/01-wazuh.conf.bak
           # curl -so /etc/logstash/conf.d/01-wazuh.conf https://raw.githubusercontent.com/wazuh/wazuh/v|WAZUH_LATEST|/extensions/logstash/6.x/01-wazuh-local.conf
-          # usermod -a -G ossec logstash
+          # usermod -a -G wazuh logstash
```

Comment:
I'd revert this change since this guide is to upgrade Elastic from 6.x to 6.8. Those versions don't support Wazuh v4.2 so people following this guide have Wazuh versions with the `ossec` user.